packages/components/src/actions/ActionList/index.tsx

```
@@ -44,6 +44,7 @@ export interface IActionListItemProps {
   description?: string;
   destructive?: boolean;
   onPress?: (close: () => void) => void | Promise<boolean | void>;
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)